source/jobs/recover-jobs.txt

```diff
@@ -0,0 +1,77 @@
+.. _rm-sync-jobs-recoverability:
+
+==================
+Recover a Sync Job
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Relational Migrator can attempt to recover continuous (CDC) sync jobs 
+that have failed within the preceding 24-hour period. 
+The procedure for recovering a CDC sync job differs, 
+depending on whether you're using the embedded or Kafka deployment 
+mode. For details on each job type, see :ref:`rm-sync-jobs`.
+
+About this Task
+---------------
+
+CDC sync job recovery depends on the type of deployment 
+mode your using. The following table summarizes the modes:
```

Comment:
(nit -- punctuation)
I think this is subjective. Because you're introducing the table, the colon isn't wrong. I have a slight preference for a period here. The reasoning is that the table intro sentence is itself a complete sentence, so no need for the colon.

Reply:
I will update for both occurrences.